jmclient/jmclient/taker.py

```diff
@@ -287,7 +286,8 @@ def prepare_my_bitcoin_data(self):
             #to make failures unlikely while keeping the occurence of failure to
             #find sufficient utxos extremely rare. Indeed, a doubling of 'normal'
             #txfee indicates undesirable behaviour on maker side anyway.
-            self.total_txfee = 2 * self.txfee_default * self.n_counterparties
+            self.total_txfee = estimate_tx_fee(3, 2,
+                txtype=self.wallet.get_txtype()) * self.n_counterparties
```

Comment:
3 inputs and 2 outputs per cp seems reasonable for an initial estimate; but why put the multiplication outside the fee estimator? the difference is small of course (version, locktime etc.), so it doesn't matter, I'd just like to know why you did it that way.

Reply:
Hi Adam, basically this was to be on the conservative side for this estimate, see the comment directly above the line:

```
adjust the required amount upwards to anticipate an increase in transaction fees after re-estimation; this is sufficiently conservative to make failures unlikely while keeping the occurence of failure to find sufficient utxos extremely rare
```

So instead of estimating 2 IN, 2 OUT I've chosen 3 IN to be absolutely sure that we dont run into too few funds problems later on when the actually needed fees are known.